types/jsonp-promise/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for jsonp-promise 0.1
+// Project: https://github.com/alexbardas/jsonp-promise
+// Definitions by: Phi <https://github.com/cdpark0530>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = jsonp;
+
+// tslint:disable-next-line no-unnecessary-generics
+declare function jsonp<R = any>(url: string, options: JsonpOptions): JsonpResult<R>;
```

Comment:
```suggestion
declare function jsonp<R = any>(url: string, options?: JsonpOptions): JsonpResult<R>;
```
options are fully optional IMO:
https://github.com/alexbardas/jsonp-promise/blob/master/index.js#L22


Aso, 'any' is a proper return type e.g. when using `JSON.parse(...)`, but here it's not very
usefull, as it accepts simple types, like `string` or `null`. `unknown` would be more proper for modern TS. Also it can be written quickly that way to reflect that return type is always an object when using JSONP (or in some similar notation)
```ts
declare function jsonp<R extends {} = Record<string, unknown>>(url: string, options?: JsonpOptions): JsonpResult<R>;
```
(the model has to extend an object, as this is what JSONP return to the calling function)